### DISCUSSION TITLE
SNP filtering criteria

### DISCUSSION THREAD

**Discussion by iranmdl** [PGScatalog/pgsc_calc#201] [Q&A] [answered]
Hi!
I have just started to play with pgs_calc tool (I am also very new to the PRS world) , and I'd like to know if there is any documentation about the specific (if any) filters applied to the target/base data. For example, if the SNPs are filtered by MAF (--maf), missing call rates (--geno), LD (--indep-pairwise), imputation quality.. etc.

Thank you in advance!

**Comment by smlmbrt** [comment, marked as answer]
Hi @iranmdl, we don't do any filtering of the genotype data at the moment - we currently assume that the data you submit has already been QC'd. 

We only applying filtering to the reference panel variants in order to generate the PCA ([see docs](https://pgsc-calc.readthedocs.io/en/latest/explanation/geneticancestry.html#implementation-of-ancestry-steps-within-pgsc-calc)), and these parameters can be edited in the `nextflow.config` file and viewed in the docs [here](https://pgsc-calc.readthedocs.io/en/latest/reference/params.html#schema).

**Reply by iranmdl** [reply]
Hi Sam!
Thank you for the very quick response 😊.
I am trying to compute PRS scores using UK biobank data, and I stumbled upon the workflow suggested [here](https://2cjenn.github.io/PRS_Pipeline/), where they apply filters such as `--maf` or imputation quality. And also they restrict the analysis to individuals who: have missing rate in autosomes ≤ 0.02, are not outliers for missingness or heterozygosity, are in a maximal set of unrelated individuals, are not sex-discordant. So I wondered first if `pgsc_calc` applies any of this (now I know it does not), or if I should apply those filters myself before running `pgsc_calc`? The idea is to calculate PRSs in UKB data using PGS catalog.

Thank you again 🙏

**Reply by smlmbrt** [reply]
Yes, if you would like to apply those filters to the samples and variants that you want to be eligible for PGS calculation then we suggest you do that prior to running the pipeline. Wether or not to include related samples really depends on your use-case, but often matters more at the stage of PGS development or evaluation (because it could over-estimate performance).
